scripts/seed.ts

```diff
@@ -0,0 +1,58 @@
+import { PrismaClient } from '@prisma/client';
+import recipes from './recipes.json';
+
+const prisma = new PrismaClient();
+
+const seed = async () => {
+  console.log('seeding started');
+
+  // cleanup
+  await Promise.all([
+    prisma.app_user.deleteMany(),
+    prisma.recipe.deleteMany(),
+    prisma.ingredient.deleteMany(),
+    prisma.direction.deleteMany(),
+    prisma.userFavorites.deleteMany(),
+  ]);
+
+  // pre-populate users from auth?
+
+  const user = await prisma.app_user.create({
+    data: {
+      displayName: 'adminGuy123',
+    },
+  });
+
+  await Promise.all(
+    recipes.map((r) =>
+      prisma.recipe.create({
```

Comment:
example of a single prisma create statement that has nested create statements inside for the ingredients and directions.